src/learning-catalogue/model/factory/audienceFactory.ts

```diff
@@ -0,0 +1,23 @@
+import {Audience} from '../audience'
+import * as moment from 'moment'
+
+export class AudienceFactory {
+	constructor() {
+		this.create = this.create.bind(this)
```

Comment:
This line shouldn't be needed?

Reply:
Unfortunately 'this' is undefined when the create method is called from a Promise. This fixed the problem. I'm happy to change if there's a better solution.